Dockerfile

```diff
@@ -103,6 +106,7 @@ ENV PATH="/root/.cargo/bin:$PATH"
 
 RUN --mount=type=cache,id=apt-$TARGETARCH$TARGETVARIANT,sharing=locked,target=/var/cache/apt \
     apt-get update && \
+    apt-get dist-upgrade && \
```

Comment:
This is done in https://github.com/goauthentik/fips as one of the final steps, so we can save time here.